src/app/frontend/common/components/sparkline/sparkline.html

```diff
@@ -0,0 +1,8 @@
+<svg viewBox="0,0 1,1"
```

Comment:
Nit: please add the ` Copyright ...` boilerplate to html files. See how it is done in others. 

No worries, it is stripped and minified during build process.